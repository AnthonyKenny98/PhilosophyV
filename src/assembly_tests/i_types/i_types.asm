nop
addi $x1, $x0, 1 	# $x1 = 1
addi $x2, $x1, 1	# $x2 = 2
ori $x3, $x2, 1 	# $x3 = 3
ori $x5, $x1, 4	    # $x5 = 5
andi $x4, $x5, 6	# $x4 = 4
addi $x7, $zero, 7 	# $x7 = 7
addi $x6, $x7, -1 	# $x6 = 6
slti $x8, $x6, 7 	# $x8 = 1
slti $x9, $x6, 5 	# $x9 = 0
slti $x10, $x6, 6 	# $x10 = 0
sltiu $x11, $x0, 1  # $x11 = 1
xori $x12, $x7, 5 	# #x12 = 2