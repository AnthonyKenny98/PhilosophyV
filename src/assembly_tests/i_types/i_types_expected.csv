32
0
1
2
3
4
5
6
7
1
0
0
1
2
13
3
-8
-2
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0